epub33/rs/index.html

```diff
@@ -384,14 +384,18 @@ <h3>Network access</h3>
 			<section id="sec-epub-rs-external-links">
 				<h3>External links</h3>
 
-				<p data-tests="#pub-external-links">Reading systems SHOULD open links that resolve outside the <a>EPUB
-						publication</a> in a new browser instance to ensure that the browser's security and privacy
-					controls are available to users.</p>
+				<p>Reading systems SHOULD alert users before opening hyperlinks that declare a <a
+						data-cite="url#concept-url-scheme">scheme</a> [[url]] (i.e., that resolve outside the [=EPUB
+					publication=] or require a specialized application to open).</p>
+
+				<p data-tests="#pub-external-links">Reading systems SHOULD open links that declare a scheme in an
+					appropriate application to ensure security and privacy controls are available.</p>
```

Comment:
(This is a matter of taste:) I wonder if it does not read better to merge the two paragraphs into one.

Reply:
I didn't want to interfere with the existing test, but I guess that could be shuffled to a span, right?